metadata.rb

```diff
@@ -9,7 +9,7 @@
 issues_url 'https://github.com/cvent/octopus-deploy-cookbook/issues'
 version '0.12.0'
 
-depends 'windows', '>= 1.38.0'
+depends 'windows', '>= 3.0.5'
```

Comment:
I actually think we do not need the cookbook anymore.  The main reason for having it was simply for backwards compatibility issues.

Reply:
Ahh forgot about the tools resource which does require it.  I would prefer to just not specify a version here TBH and make it so its picked outside of this cookbook.

Reply:
Sever and tentacle don't, but it looks like tools still uses the `windows_zipfile` resource which I believe does require the windows cookbook. 

https://github.com/cvent/octopus-deploy-cookbook/blob/master/resources/tools.rb#L44-L48

Reply:
I am not sure if the version >= 1.38.0  was needed or not, One thing you could do is add a dependency on windows >= 3.0.5 in the berksfile so that in testing it would use that version.